x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_landing.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  EuiButton,
+  EuiLink,
+  EuiPageHeader,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { getAppSearchUrl } from '../../../shared/enterprise_search_url';
+import { ENGINE_CRAWLER_PATH } from '../../routes';
+import { generateEnginePath } from '../engine';
+
+import './crawler_landing.scss';
+import { CRAWLER_TITLE } from '.';
+
+export const CrawlerLanding: React.FC = () => (
+  <div data-test-subj="CrawlerLanding" className="crawler-landing">
+    <EuiPageHeader pageTitle={CRAWLER_TITLE} />
+    <EuiSpacer />
+    <EuiPanel grow paddingSize="l" className="crawler-landing__panel">
+      <div>
+        <EuiTitle size="s">
+          <h1>
+            {i18n.translate('xpack.enterpriseSearch.appSearch.engine.crawler.landingPage.title', {
+              defaultMessage: 'Setup the Web Crawler',
+            })}
+          </h1>
+        </EuiTitle>
+        <EuiSpacer />
+        <EuiText>
+          <p>
+            {i18n.translate(
+              'xpack.enterpriseSearch.appSearch.engine.crawler.landingPage.description',
+              {
+                defaultMessage:
+                  "Easily index your website's content. To get started, enter your domain name, provide optional entry points and crawl rules, and we will handle the rest.",
+              }
+            )}{' '}
+            <EuiLink
+              external
+              target="_blank"
+              href="https://www.elastic.co/guide/en/app-search/current/web-crawler.html"
+              data-test-subj="CrawlerDocumentationLink"
+            >
+              {i18n.translate(
+                'xpack.enterpriseSearch.appSearch.engine.crawler.landingPage.documentationLinkLabel',
+                {
+                  defaultMessage: 'Learn more about the web crawler',
```

Comment:
Copy nit: does this need a full stop at the end of the sentence/paragraph?
```suggestion
                  defaultMessage: 'Learn more about the web crawler.',
```

Reply:
I've fixed this in 47c3823